src/Components/Server/src/Circuits/CircuitRegistry.cs

```diff
@@ -391,17 +393,18 @@ private static class EventIds
                 public static readonly EventId ExceptionDisposingCircuit = new EventId(100, "ExceptionDisposingCircuit");
                 public static readonly EventId ExceptionDisposingTokenSource = new EventId(101, "ExceptionDisposingTokenSource");
                 public static readonly EventId AttemptingToReconnect = new EventId(102, "AttemptingToReconnect");
-                public static readonly EventId InvalidCircuitId = new EventId(103, "InvalidCircuitId");
-                public static readonly EventId ConnectingToActiveCircuit = new EventId(104, "ConnectingToActiveCircuit");
-                public static readonly EventId ConnectingToDisconnectedCircuit = new EventId(105, "ConnectingToDisconnectedCircuit");
-                public static readonly EventId FailedToReconnectToCircuit = new EventId(106, "FailedToReconnectToCircuit");
-                public static readonly EventId CircuitDisconnectStarted = new EventId(107, "CircuitDisconnectStarted");
-                public static readonly EventId CircuitNotActive = new EventId(108, "CircuitNotActive");
-                public static readonly EventId CircuitConnectedToDifferentConnection = new EventId(109, "CircuitConnectedToDifferentConnection");
-                public static readonly EventId CircuitMarkedDisconnected = new EventId(110, "CircuitMarkedDisconnected");
-                public static readonly EventId CircuitEvicted = new EventId(111, "CircuitEvicted");
-                public static readonly EventId CircuitDisconnectedPermanently = new EventId(112, "CircuitDisconnectedPermanently");
-                public static readonly EventId CircuitExceptionHandlerFailed = new EventId(113, "CircuitExceptionHandlerFailed");
+                public static readonly EventId InvalidCircuitSecret = new EventId(103, "InvalidCircuitSecret");
+                public static readonly EventId FailedToFindCircuit = new EventId(104, "FailedToFindCircuit");
```

Comment:
There's a cascade of updates because I wanted to try and keep the logs in some kind of reasonable order based on their usage.